tests/nnapi/specs/V1_0/l2_normalization_quant8.mod.py

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (C) 2017 The Android Open Source Project
+#
```

Comment:
Please add `Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved` too.

Reply:
Please change test spec file name if it is not original code from android: `l2_normalization_quant8.mod.py` -> `l2_normalization_quant8_nnfw.mod.py`

Reply:
Thank you for the comments. I have made the necessary changes.

Reply:
@hasw7569 PTAL